templates/data-sources/ip_ranges.md.tmpl

```diff
@@ -1,4 +1,4 @@
-{{define "fastly_ip_ranges"}}---
+{{define "ip_ranges"}}---
```

Comment:
NOTE: this was originally named `fastly_ip_ranges` because of the original `parse-templates.go` script which tried to generate a 'side menu' and was doing so based off prefixing 'fastly' into a 'Title' field. 

In the new script that generates the documentation this function no longer exists. **Is that a problem?**

```go
func generateMenuItems(pageType string, pages []Page) []MenuItem {
	var menuItems []MenuItem
	for _, p := range pages {
		menuItems = append(menuItems, MenuItem{
			Lookup: fmt.Sprintf("docs-fastly-resource-%s", strings.ReplaceAll(p.name, "_", "-")),
			Link:   fmt.Sprintf("/docs/providers/fastly/%s/%s.html", pageType, p.name),
			Title:  fmt.Sprintf("fastly_%s", p.name),
		})
	}
	return menuItems
}
```

Reply:
@phamann I don't think this part of the original script was actually used in the latest documentation because the links in the side menu of the current documentation ([here](https://registry.terraform.io/providers/fastly/fastly/latest/docs)) are `/docs/resources/service_acl_entries_v1` and **_not_** `/docs/providers/fastly/%s/%s.html`.

This makes me wonder if the side menu is now dynamically generated by Hashicorp. 

## UPDATE

That does appear to be the case when reading https://www.terraform.io/docs/registry/providers/docs.html#navigation-hierarchy

> Navigation Hierarchy: Provider docs are organized by category: resources, data sources, and guides. At a minimum, a provider must contain an index (docs/index.md) and at least one resource or data source. After publishing this provider version, its page on the Terraform Registry would display a navigation which resembles this hierarchy.

...all of which we have.

So I'm not going to put back that logic as it now appears to be confirmed as redundant/obsolete.